### DISCUSSION TITLE
Spacing for bars after text

### DISCUSSION THREAD

**Discussion by kaboddy** [lptstr/winfetch#138] [Q&A] [answered]
Just curious if there's a way to line up the output of the bars beside the text for drives?  Mine is sort of shifty, and my OCD is getting crazy. ;)

![image](https://user-images.githubusercontent.com/69687422/170391639-58993232-e7f1-4eab-a777-5703cacdf602.png)

**Comment by rashil2000** [comment, marked as answer]
I'm afraid there's no way to do that currently 😅

**Reply by kaboddy** [reply]
I ended up thinking about tabs and how that would help.  It worked for all of them but one, so I just cheated and added some spaces...
```
'textbar' { return "$text  `t$(get_percent_bar $percentage)" }
```

Does the trick for my PC anyway.  Here's an example from another PC.
![image](https://user-images.githubusercontent.com/69687422/170487145-cc653e19-4b59-4455-837e-6431d24e5404.png)

**Reply by rashil2000** [reply]
Interesting, I mean you can always write a custom function to return anything you want, and I'm glad to see that being used well :)

**Reply by kaboddy** [reply]
Yeah I was thinking of counting returned characters and spacing based on that with if conditions.  I'm a bit of a newb with PowerShell, but am addicted a bit.  Thanks for the script!
